spyder/config/main.py

```diff
@@ -81,6 +81,7 @@
               'use_custom_cursor_blinking': False,
               'show_internal_errors': True,
               'check_updates_on_startup': True,
+              'startup_in_debug_mode': False,

```

Comment:
There's no need of this. Please remove it.

Reply:
yep, will revert cli option, add restart action to Tools, and use environment variable in restart.py